package.json

```diff
@@ -31,28 +34,24 @@
   ],
   "scripts": {
     "audit": "lerna run audit --stream",
-    "bootstrap": "lerna bootstrap",
-    "bump": "lerna publish",
     "commit": "cz",
     "doctoc": "doctoc README.md packages/collector/README.md",
     "lint": "node misc/is_eol || (eslint .*.js *.js && npm run typecheck && lerna run lint --stream && eslint misc && eslint dockerfile-examples && eslint bin)",
-    "postinstall": "husky install && npm run bootstrap",
+    "postinstall": "husky install && lerna run node_modules:exists",
     "precommit": "npm run doctoc",
     "coverage-all": "nyc --reporter=lcov lerna run test --stream",
     "coverage": "nyc --reporter=lcov lerna run test --scope=${npm_config_scope} && npm run open-coverage",
     "open-coverage": "open coverage/lcov-report/index.html",
     "prettier": "prettier --write 'misc/**/*.js' 'dockerfile-examples/**/*.js' 'bin/**/*.js' && lerna run prettier --stream",
-    "refresh-package-lock-files": "lerna exec 'rm -rf node_modules' && npm install",
     "reinstall-deps": "lerna clean -y && rm -rf node_modules && npm install",
     "test": "lerna run test --stream",
     "test:ci": "lerna run test:ci --stream",
     "test:ci:collector": "lerna run test:ci  --stream --scope=@instana/collector",
     "test:debug": "lerna run test:debug --stream",
     "typecheck": "tsc",
-    "update-deps": "lerna exec '../../node_modules/.bin/ncu --upgrade' && lerna exec 'npm i --package-lock-only' && npm run reinstall-deps",
```

Comment:
TBD: I never used this command and I don't see a use case for it. I can re-add the cmd.

Reply:
Good point 👍 Will cleanup